gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,7 +1,7 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 distributionSha256Sum=e111cb9948407e26351227dabce49822fb88c37ee72f1d1582a69c68af2e702f
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.1.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions-snapshots/gradle-8.3-20230709222231+0000-bin.zip
```

Comment:
Does this work without changing the `distributionSha256Sum` checksum?

Reply:
Nope it didn't 😁  Fixed the checksum accordingly